local-proxy/src/server.ts

```diff
@@ -21,6 +23,14 @@ if (!localToken) {
   throw new Error('SHIFT_DEV_SERVER_LOCAL_TOKEN env var not defined');
 }
 
+function setupEnv({ port }: { port: number }) {
+  process.env.DB_ENDPOINT = `http://localhost:${port}/v1`;
+  // TODO(vogre): Used (at least) for DB, where it doesn't really
+  //     matter locally.  Are better values available?
+  process.env.APP_ID = 'local-app';
+  process.env.ACCOUNT_ID = 'me';
+}
+
 const whitelistedModulesArr = (process.env.SHIFT_DEV_SERVER_MODULES_WHITELIST || '@binaris/shift-db').split(',');
```

Comment:
Note that we should not load `@binaris/shift-db` before `setupEnv`, otherwise the environment variables are read during parsing and not on the first request.

Reply:
In fact, `@binaris/shift-db` should not be whitelisted at all it's `@binaris/shift-leveldb-server` which has the exposed `poll` that's required here.